package.xml

```diff
@@ -62,77 +62,59 @@
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
     <notes>
         <![CDATA[
-### Fixes across all extensions
-- Handle Apache graceful restarts more accurately #2483
-- Mark config as immutable and interned to avoid refcounting race conditions #2516
-
 ## Tracer
-This release includes Windows dlls for the tracer starting with PHP 7.2. #2230
 
-Windows support is in beta. We recommend carefully deploying the tracer to staging environments and testing it there before putting it to production. We are looking towards stabilizing Windows quickly, so any feedback and issue reports are welcome.
+Windows support is officially GA (generally available) and considered stable.
+
+Some performance work has been undertaken to primarily reduce the overhead of telemetry reporting.
```

Comment:
Can WE share a number with cautious wording?

Reply:
It depends on things, like composer.json size and depends on what your app is actually doing, like if there would be a lot of logs or not if logging would be fully turned on.

For the changes on the ddtrace-1.0.0 branch we probably can have more concrete perf numbers as it's a rather overall startup improvement.